hazelcast/discovery.py

```diff
@@ -58,11 +58,11 @@ def refresh(self):
         try:
             self._private_to_public = self.cloud_discovery.discover_nodes()
         except Exception as e:
-            _logger.warning("Failed to load addresses from Hazelcast.cloud: %s", e)
+            _logger.warning("Failed to load addresses from Hazelcast Viridian: %s", e)
 
 
 class HazelcastCloudDiscovery:
```

Comment:
We can't change that, as it is sometimes used in the wild for monkey patching the coordinator URL. We would break people's code. The same goes for the configuration element `cloud_discovery_token`